sklearn/utils/estimator_checks.py

```diff
@@ -337,20 +359,24 @@ def _is_32bit():
     return struct.calcsize('P') * 8 == 32
 
 
-def check_estimator_sparse_data(name, Estimator):
+def check_estimator_sparse_data(name, estimator_org):
```

Comment:
As someone who has worked too much on Named Entity Recognition, I can't read "org" as anything but "organisation". Can we use `estimator_orig` or `base_estimator`?